content/applications/services/project/tasks/recurring_tasks.rst

```diff
@@ -0,0 +1,54 @@
+===============
+Recurring tasks
+===============
+
+When handling a project, the same task often needs to be performed several times: for example,
+a weekly meeting or status report. The **recurring tasks** feature allows you to automate the
+creation of those tasks.
+
+.. seealso::
+   `Odoo Tutorials: Recurring tasks <https://www.odoo.com/slides/slide/recurring-tasks-1946>`_
+
+Configuration
+=============
+
+To enable recurring tasks, go to :menuselection:`Project --> Configuration --> Settings`, then
+activate :guilabel:`Recurring Tasks`.
+
+Once configured in :guilabel:`Settings`, the recurring tasks feature is activated by default on all
+existing projects. If needed, it can be deactivated on an individual project by clicking the ...
+icon next to the project name, then going to :menuselection:`Settings --> Settings --> Task
+Management` and unchecking the :guilabel:`Recurring Tasks`.
+
+.. image:: recurring_tasks/project-settings.png
+   :align: center
+   :alt: Press the ... icon next to the project, then go to Settings
+
+Set up recurrence on a task
+---------------------------
+
+In an existing task, go to the :guilabel:`Recurrent` tab, then check the :guilabel:`Recurrent` box.
+A set of options allows you to configure the frequency (daily, monthly, weekly, or other) and the
+number of recurrences.
+
+.. important::
```

Comment:
I would just put it as a note.

Reply:
Not sure if it's better as a note, let's check together